README.md

```diff
@@ -10,6 +10,8 @@ sudo ./install.sh v1.N.x/v1.N-rcM
 The go.sh and install.sh for pre-v1.7 releases have been deleted,
 you can find them in the history.
 
+There are no branches anymore, everything is in the default branch.
```

Comment:
You mean,
There is no other branch anymore, everything is in the default(main) branch

Reply:
Rephrased, thanks.